src/dynamic_sharing_service/extension/third.party.attribute.extension.content.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const Validation = require('../../yoti_common/validation');
+
+class ThirdPartyAttributeExtensionContent {
+  constructor(expiryDate, definitions) {
+    Validation.instanceOf(expiryDate, Date, 'expiryDate');
+    Validation.hasOnlyStringValues(definitions, 'definitions');
```

Comment:
Perhaps this could be an array or `Definition`, that will serialise to `{ "name": "some name"  }` using `.toJSON()`.

This would prevent a future breaking change if we need to add a new _definition_ property.